app/src/main/java/com/es/findsoccerplayers/fragments/FragmentInfoBookedMatch.java

```diff
@@ -4,16 +4,78 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
 
+import androidx.annotation.NonNull;
 import androidx.fragment.app.Fragment;
 
+import com.es.findsoccerplayers.ActivityInfoBookedMatch;
 import com.es.findsoccerplayers.R;
+import com.es.findsoccerplayers.models.Match;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
 
 public class FragmentInfoBookedMatch extends Fragment {
 
+    private String relatedMatch;
+    private TextView field, day, time, quota, missingPlayers, description;
+    private Button retireBtn;
+    private FirebaseDatabase db;
+    private Match m;
+
+    public FragmentInfoBookedMatch(String relatedMatch){
+        super();
+        this.relatedMatch = relatedMatch;
+    }
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.frag_info_booked_match, container, false);
+
+        field = view.findViewById(R.id.frag_info_booked_match_campoTV);
```

Comment:
nomi sempre in inglese nel layout, meglio farlo subito che se no ci tocca riguardare tutto da 0 alla fine